.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+**Task name**: <Name of changed or new azure pipeline task>
```

Comment:
I would remove "azure" from the sentence

Reply:
Fixed it 👍